packages/react-components/react-nav-preview/library/src/components/SplitNavItem/useSplitNavItemStyles.styles.ts

```diff
@@ -20,45 +21,37 @@ export const splitNavItemClassNames: SlotClassNames<SplitNavItemSlots> = {
  */
 const useSplitNaveItemStyles = makeStyles({
   baseRoot: {
+    display: 'flex',
     gap: 'unset',
-    alignItems: 'start',
+    alignItems: 'stretch',
     padding: 'unset',
     textAlign: 'unset',
     backgroundColor: navItemTokens.backgroundColor,
+    ...navItemTokens.transitionTokens,
 
-    transitionDuration: navItemTokens.animationTokens.animationDuration,
-    transitionTimingFunction: navItemTokens.animationTokens.animationTimingFunction,
-    transitionProperty: 'background',
-
-    ':hover .fui-NavItem': {
-      backgroundColor: navItemTokens.backgroundColorHover,
-      transitionDuration: navItemTokens.animationTokens.animationDuration,
-      transitionTimingFunction: navItemTokens.animationTokens.animationTimingFunction,
-      transitionProperty: 'background',
-    },
-
-    ':active .fui-NavItem': {
-      backgroundColor: navItemTokens.backgroundColorPressed,
-      transitionDuration: navItemTokens.animationTokens.animationDuration,
-      transitionTimingFunction: navItemTokens.animationTokens.animationTimingFunction,
-      transitionProperty: 'background',
-    },
+    ':hover .fui-NavItem': navItemTokens.transitionTokens,
+    ':active .fui-NavItem': navItemTokens.transitionTokens,
+    ':hover .fui-NavSubItem': navItemTokens.transitionTokens,
+    ':active .fui-NavSubItem': navItemTokens.transitionTokens,
   },
-  basenavItem: {
+  baseNavItem: {
     // styles that we want to disagree with the default on
     display: 'flex',
     textTransform: 'none',
+    alignSelf: 'stretch',
     textAlign: 'left',
     position: 'relative',
     justifyContent: 'start',
     gap: tokens.spacingVerticalL,
   },
   baseSecondary: {
-    maxWidth: '24px',
-    minWidth: '24px',
-    paddingInline: '4px',
-    marginBlockStart: '4px',
-
+    maxWidth: '28px',
+    minWidth: '28px',
+    paddingInlineEnd: '12px',
+    paddingInlineStart: '5px',
```

Comment:
I see these changed from 4px -> 5px, which deviates from our token set, can we set back to 4px and use spacing token versions?

Reply:
Yeah - this stuff is all bispoke to line up nicely under the carrot icon fron NavCategoryItem.